howto/android/graphics-debugging-with-renderdoc.md

```diff
@@ -0,0 +1,33 @@
+[Renderdoc](https://github.com/baldurk/renderdoc) is a graphics frame debugger which supports Android applications.
+
+Debugging graphics can be challenging and [Renderdoc](https://github.com/baldurk/renderdoc) is a great tool to provide insight into Vulkan and OpenGL command execution. The following describes how you can use Renderdoc with Anbox Cloud.
+
+## Install Renderdoc
+
+Renderdoc provides binary builds [here](https://renderdoc.org/builds). These come with all prerequisites to be able to capture on Android.
+
+For capturing Android applications you also need the Android SDK as Renderdoc needs a few tools like adb or aapt. The easiest is to install [Android Studio](https://developer.android.com/studio) which will allow you to install the SDK.
```

Comment:
Shouldn't we instead tell people to install from the snap here? We could link to the snapstore https://snapcraft.io/android-studio, or use the CLI to install:
```sh
sudo snap install android-studio --classic
```

What do you think?

Reply:
No, we should only ever point to the official installation instructions of external tools we point to and do not reference custom packaged, not official ways of installing.